dev/how-to/upgrade/from-previous-version.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: TiDB 3.0 升级操作指南
 category: how-to
+aliases: ['/docs-cn/op-guide/tidb-v3.0-upgrade-guide/']
```

Comment:
```suggestion
```
We only need `aliases` in `v3.0`. Remember to remove this line when you copy the doc from v3.0.